src/content/i18n/de.yml

```
@@ -64,6 +65,8 @@ starlight.title: Willst du deine eigene Dokumentation erzeugen?
 starlight.description: Mit dieser Vorlage kannst du loslegen.
 # `<StudioHeading>` component
 studioHeading.label: Studio-Funktion
+# Upgrade guide
+upgrade.implementationPR: 'Implementation PR: '
```

Ich würde vorschlagen, dass wir hier eher das Wort "Implementierung" verwenden und einen Bindestrich einfügen!

```suggestion
upgrade.implementationPR: 'Implementierungs-PR: '
```